srcpkgs/jamulus/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'jamulus'
+pkgname=jamulus
+version=3.6.2
+revision=1
+archs="x86_64* i686"
+wrksrc=Jamulus${version}
+build_style=qmake
+configure_args="Jamulus.pro"
+hostmakedepends="pkg-config"
+makedepends="qt5-declarative-devel jack-devel qjackctl"
```

Comment:
`qjackctl` is makedepends?

Reply:
It was listed as part of the build process [here](https://jamulus.io/wiki/Installation-for-Linux), I could try moving it to `depends` instead if that's more appropraite.

Reply:
I see what you mean. After reading through the jamulus documentation more, a user would just install the `qjackctl` package on their own if that's how they wanted to set up jack with it. Removed `qjackctl` from `makedepends`.